packages/website/pages/api/addTags.js

```diff
@@ -0,0 +1,37 @@
+import mailchimp from '@mailchimp/mailchimp_marketing'
+
+const LIST_ID = '64f6e3fd11'
+
+/**
+ * Subscribe user to blog mailing list
+ * @param {import('next').NextApiRequest} req
+ * @param {import('next').NextApiResponse} res
+ */
+export default async (req, res) => {
```

Comment:
can you please fix the linter warnings please ?

Reply:
next api routes have been removed